src/features/scheduling/generation/SessionScheduler.ts

```
@@ -17,9 +18,226 @@ export class SessionScheduler {
+        // Create NightShift object for this bunk
+        shift[bunkNumber] = {
+          counselorsOnDuty: [],
+          nightBunkDuty: [],
+        };
+
+        // choosing 4 employees
+        const employeesInBunk = employees.filter((e) => e.bunk === bunkNumber);
+        const PER_SPLIT = Math.ceil(employeesInBunk.length / 2);
+
+        if (employeesInBunk.length === 0) continue;
+        let selected: StaffAttendeeID[] = []
```

ESLint

[error] 'selected' is never reassigned. Use 'const' instead.

(prefer-const)